r-package/grf/R/tuning.R

```diff
@@ -66,11 +68,13 @@ tune_forest <- function(data,
   variance.guess <- rep(var(small.forest.errors) / 2, nrow(fit.draws))
   kriging.model <- tryCatch({
     capture.output(
-      model <- DiceKriging::km(
-        design = data.frame(fit.draws),
-        response = small.forest.errors,
-        noise.var = variance.guess
-      )
+      with_seed(
+        model <- DiceKriging::km(
```

Comment:
From the link you provided it seems randomness in DiceKringing arises when `parinit` is not provided, would an alternative be to find the equivalent value to provide for that instead? (though I guess either way is fine)

(https://www.rdocumentation.org/packages/DiceKriging/versions/1.5.5/topics/km)

Reply:
Left as is -- let me know if you change your mind about either way being fine.